v20.2/create-user.md

```diff
@@ -1,30 +1,34 @@
 ---
 title: CREATE USER
-summary: The CREATE USER statement creates SQL users, which let you control privileges on your databases and tables.
+summary: The CREATE USER statement creates SQL roles that are preconfigured with the necessary login permission to act as database users.
 toc: true
 ---
 
-The `CREATE USER` [statement](sql-statements.html) creates SQL users, which let you control [privileges](authorization.html#assign-privileges) on your databases and tables.
+The `CREATE USER` [statement](sql-statements.html) creates a SQL role that can be used to log in to a database.
+
+ There is no distinct "user" entity in CockroachDB. A role with the [`LOGIN`](#parameters) option enabled can log in to the SQL shell and is often called a user.
+ 
+ You can [assign privileges](authorization.html#assign-privileges) to the user and [set it as a member](grant-roles.html) of other roles, inheriting their privileges.
 
 {{site.data.alerts.callout_info}}
- Since the keywords `ROLE` and `USER` can now be used interchangeably in SQL statements for enhanced Postgres compatibility, `CREATE USER` is now an alias for [`CREATE ROLE`](create-role.html).
+ `CREATE USER` is equivalent to the statement `CREATE ROLE`, with one exception: `CREATE ROLE` sets the [`NOLOGIN`](#parameters) option by default, preventing the new role from being used to log in to the database. You can use `CREATE ROLE` and specify the [`LOGIN`](#parameters) option to achieve the same result as `CREATE USER`.
+
+ Prior to CockroachDB v20.1, role-based access control (RBAC) was an enterprise feature, and `CREATE ROLE` created an entity type called a role that was distinct from a user. As of v20.1, an updated version of this feature is freely available in CockroachDB core, and for enhanced Postgres compatibility, the keywords `ROLE` and `USER` can now be used interchangeably in SQL statements to refer to this new, singular entity type.
 {{site.data.alerts.end}}
 
+For more information, see the [Authorization](authorization.html) documentation on roles and users.
```

Comment:
ditto hyperlinks

Reply:
Updated the reference to the Authorization page; saving the second for the follow-up issue.